examples/frame_vlmetalayers.c

```diff
@@ -51,36 +51,53 @@ int main(void) {
 
 
   // Add a metalayer
-  int vlmetalater_len = blosc2_meta_add(schunk, "vlmetalayer", (uint8_t *) "This is a vlmetalayers content...", 10);
-  if (vlmetalater_len < 0) {
-    printf("Cannot write vlmetalayers chunk");
-    return vlmetalater_len;
+  int metalater_len = blosc2_meta_add(schunk, "metalayer", (uint8_t *) "This is a metalayers content...", 10);
```

Comment:
metalater_len -> metalayer_len  (although metalater_cocrodile came to my mind too ;-)